tests/pages/services/ServiceActions-cy.js

```diff
@@ -138,7 +128,7 @@ describe('Service Actions', function () {
             response: {
               message: {message: 'Not Authorized to perform this action!'}
             },
-            delay: 50
+            delay: 100
```

Comment:
Unfortunately I couldn't get the test to pass without increasing this delay. @rcorral and I briefly spoke about it yesterday and concluded that it's probably a poorly written test. He mentioned a method of chaining Cypress methods that should allow us to remove these delays throughout the tests — @rcorral perhaps this is something we should do now?